modules/KoreBuild.Tasks/PackNuSpec.cs

```diff
@@ -14,22 +14,59 @@
 
 namespace KoreBuild.Tasks
 {
+    /// <summary>
+    /// Generates a nupkg from a nuspec file
+    /// </summary>
     public class PackNuSpec : Microsoft.Build.Utilities.Task
     {
+        /// <summary>
+        /// The path the nuspec file.
+        /// </summary>
         [Required]
         public string NuspecPath { get; set; }
 
-        [Required]
+        /// <summary>
+        /// Output nupkg is placed in folder + '$(id).$(version).nupkg'.
+        /// Either this or <see cref="OutputPath" /> must be specified.
+        /// </summary>
         public string DestinationFolder { get; set; }
```

Comment:
Maybe I'm still waking up, but this seems unclear on the difference between `DestinationFolder` and `OutputPath`. is it DestinationFolder -> `$(DestinationFolder)\$(id).$(version).nupkg\$(nupkgName)` OutputPath -> `$(OutputPath)\$(nupkgName)`? What scenarios would make someone want to pick one over the other?

Reply:
```xml
<PackNuspec DestinationFolder="artifacts\build\" /> 
<!-- Produces: artifacts\build\PackageId.PackageVersion.nupkg -->

<PackNuspec OutputPath="artifacts\build\whateverfilenameiwant.nupkg" />
<!-- Produces: artifacts\build\whateverfilenameiwant.nupkg-->
```

Reply:
To further clarify: you would want to use OutputPath when producing a nupkg that shouldn't have the $(Version) info in the file name.